docs/multicluster/antctl.md

```diff
@@ -26,5 +26,59 @@ antctl mc get clusterset [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceimport [NAME] [-n NAMESPACE] [-o json|yaml] [-A]
 antctl mc get resourceexport [NAME] [-n NAMESPACE] [-clusterid CLUSTERID] [-o json|yaml] [-A]
 ```
-
 To see the usage examples of these commands, you may also run `antctl mc get [subcommand] --help`.
+
+## antctl mc create
+
+`antctl mc create` command can create access-token and other resources like ClusterSet, ClusterClaims for 
+Antrea Multi-cluster setup
+
++ `antctl mc create accesstoken` command can create accesstoken for member clusters. 
++ `antctl mc create clusterclaims` command can create two ClusterClaims in a leader or member cluster. One for the leader or member cluster, and another for the ClusterSet.
++ `antctl mc create clusterset` command can create a ClusterSet in a leader or member cluster.
+
+```bash
+$ anctcl mc create accesstoken [NAME] [-n NAMESPACE] [--serviceaccount SERVICE_ACCOUNT] [--role-binding ROLE_BINDING]
+$ antctl mc create clusterclaims [-n NAMESPACE] [--clusterset-id CLUSTERSET_ID] [--cluster-id CLUSTER_ID]
+$ antctl mc create clusterset [NAME] [-n NAMESPACE] [--leader-server LEADER_SERVER] [--service-account SERVICE_ACCOUNT] [--secret SECRET] [--leader-cluster LEADER_CLUSTER_ID]
+```
+To see the usage examples of these commands, you may also run `antctl mc create [subcommand] --help`.
+
+## antctl mc add
+
+`antctl mc add` command can add a new member cluster to a ClusterSet.
+
+```bash
+$ antctl mc add member-cluster [CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET] [--service-account SERVICE_ACCOUNT]
+```
+
+To see the usage examples of these commands, you may also run `antctl mc add [subcommand] --help`.
+
+## antctl mc delete
+
+`antctl mc delete` command can delete resources in an Antrea Multi-cluster ClusterSet.
+
++ `antctl mc delete clusterclaims` command can delete the two ClusterClaims in a specified Namespace. One for the leader or member cluster, and another for the ClusterSet.
++ `antctl mc delete clusterset` command can delete a ClusterSet in a leader or member cluster.
++ `antctl mc delete member-cluster` command can delete a member cluster in a specified Antrea Multi-cluster ClusterSet.
+
+```bash
+$ antctl mc delete clusterclaims [-n NAMESPACE]
+$ antctl mc delete clusterset [NAME] [-n NAMESPACE]
+$ antctl mc delete member-cluster [MEMBER_CLUSTER_ID] [-n NAMESPACE] [--clusterset CLUSTERSET]
+```
+To see the usage examples of these commands, you may also run `antctl mc delete [subcommand] --help`.
+
+## antctl mc deploy
+
+`antctl mc deploy` command can deploy Antrea Multi-cluster Controller to a leader or member cluster. 
+
++ `antctl mc deploy leader-cluster` command can deploy Antrea Multi-cluster Controller to a leader cluster, and create all the CRDs leader cluster needed automatically.
```

Comment:
```suggestion
+ `antctl mc deploy leader-cluster` command can deploy Antrea Multi-cluster Controller to a leader cluster, and create all the CRDs the leader cluster needed automatically.
```

Reply:
create -> import

Reply:
Remove "automatically". All changes by a command are automatic..

Reply:
There `deploy/leader_cluster.go` used "created". I changed it to "define" too.